shared-bindings/supervisor/Runtime.c

```diff
@@ -134,12 +137,93 @@ MP_PROPERTY_GETSET(supervisor_runtime_autoreload_obj,
+//|     the services used for it."""
+//|
+STATIC mp_obj_t supervisor_runtime_get_ble_workflow(mp_obj_t self) {
+    #if CIRCUITPY_BLE_FILE_SERVICE && CIRCUITPY_SERIAL_BLE
+    return mp_obj_new_bool(supervisor_bluetooth_workflow_is_enabled());
+    #else
+    return mp_const_false;
+    #endif
+}
+MP_DEFINE_CONST_FUN_OBJ_1(supervisor_runtime_get_ble_workflow_obj, supervisor_runtime_get_ble_workflow);
+
+STATIC mp_obj_t supervisor_runtime_set_ble_workflow(mp_obj_t self, mp_obj_t state_in) {
+    #if CIRCUITPY_BLE_FILE_SERVICE && CIRCUITPY_SERIAL_BLE
+    if (mp_obj_is_true(state_in)) {
+        supervisor_bluetooth_enable_workflow();
+    } else {
+        supervisor_bluetooth_disable_workflow();
+    }
+    #else
+    mp_raise_NotImplementedError(NULL);
+    #endif
+    return mp_const_none;
+}
+MP_DEFINE_CONST_FUN_OBJ_2(supervisor_runtime_set_ble_workflow_obj, supervisor_runtime_set_ble_workflow);
+
+MP_PROPERTY_GETSET(supervisor_runtime_ble_workflow_obj,
+    (mp_obj_t)&supervisor_runtime_get_ble_workflow_obj,
+    (mp_obj_t)&supervisor_runtime_set_ble_workflow_obj);
+
+//|     next_stack_limit: int
+//|     """The size of the stack for the next vm run. If its too large, the default will be used."""
+//|
+STATIC mp_obj_t supervisor_runtime_get_next_stack_limit(mp_obj_t self) {
+    return mp_obj_new_int(get_next_stack_size());
+}
+MP_DEFINE_CONST_FUN_OBJ_1(supervisor_runtime_get_next_stack_limit_obj, supervisor_runtime_get_next_stack_limit);
+
+STATIC mp_obj_t supervisor_runtime_set_next_stack_limit(mp_obj_t self, mp_obj_t size_obj) {
+    mp_int_t size = mp_obj_get_int(size_obj);
+    mp_arg_validate_int_min(size, 256, MP_QSTR_size);
+    set_next_stack_size(size);
+    return mp_const_none;
+}
+MP_DEFINE_CONST_FUN_OBJ_2(supervisor_runtime_set_next_stack_limit_obj, supervisor_runtime_set_next_stack_limit);
+
+MP_PROPERTY_GETSET(supervisor_runtime_next_stack_limit_obj,
+    (mp_obj_t)&supervisor_runtime_get_next_stack_limit_obj,
+    (mp_obj_t)&supervisor_runtime_set_next_stack_limit_obj);
+
+//|     rgb_status_brightness: int
+//|     """Set brightness of status RGB LED from 0-255. This will take effect
+//|     after the current code finishes and the status LED is used to show
+//|     the finish state."""
+//|
+STATIC mp_obj_t supervisor_runtime_get_rgb_status_brightness(mp_obj_t self) {
+    return MP_OBJ_NEW_SMALL_INT(get_status_brightness());
+}
+MP_DEFINE_CONST_FUN_OBJ_1(supervisor_runtime_get_rgb_status_brightness_obj, supervisor_runtime_get_rgb_status_brightness);
+
+STATIC mp_obj_t supervisor_runtime_set_rgb_status_brightness(mp_obj_t self, mp_obj_t lvl) {
+    #if CIRCUITPY_STATUS_LED
+    // This must be int. If cast to uint8_t first, will never raise a ValueError.
+    int brightness_int = mp_obj_get_int(lvl);
+    mp_arg_validate_int_range(brightness_int, 0, 255, MP_QSTR_brightness);
+    set_status_brightness((uint8_t)brightness_int);
```

Comment:
You can use the return value from `mp_arg_validate_int_range()`. Or if you think it's clearer, you can assign the return value to a `uint8_t`, and then pass that. I'm not sure if doing the below makes the code smaller or the compiler is clever enough so it doesn't make a difference.
```suggestion
    set_status_brightness((uint8_t)mp_arg_validate_int_range(mp_obj_get_int(lvl), 0, 255, MP_QSTR_brightness));
```